CHANGELOG.md

```diff
@@ -1,6 +1,12 @@
 Change Log
 ==========
 
+## 1.1.9
+ * Fix to allow `hs filemanager fetch` to fetch all files
+ * `hs hubdb` now supports referencing tables by name or ID
+ * Add `hs create webpack-serverless`
+ * Fix running `hs watch` in CWD
```

Comment:
Might adjust to "Improve how CWD is determined when running `hs watch`" since the change is pretty subtle.